src/components/QuickId.vue

```diff
@@ -37,15 +42,31 @@
     bottom: 20px;
     left: 50%;
     transform: translateX(-50%);
+    width: calc(100% - 40px);
+    max-width: 350px;
 
-    .back {
+    .back-button {
       width: 25px;
-      height: auto;
+      height: 100%;
       background: url('/static/icons/grid.svg') no-repeat;
       background-size: contain;
-      border-right: 2px solid #f1f4f7;
-      margin-right: 15px;
-      padding-right: 15px;
+    }
+
+    .vertical-ruler {
+      margin: 0 15px;
+      height: 25px;
+      border-right: 1px solid #ffffff;
+      align-self: center;
+    }
+
+    .quick-id__chevron {
+      fill: #ffffff;
+      transform: rotate(-90deg);
+      cursor: pointer;
+    }
+
+    .quick-id__active-identity{
+      pointer-events: none;
```

Comment:
I think `cursor: default` is more obvious. Or probably we can apply `cursor: pointer` for the whole `.quick-id`.